tests/test_cli.py

```diff
@@ -84,6 +84,33 @@ def test_compress_already_compressed_force():
     assert h1 != h2
 
 
+def test_compress_correct_offset():
+    """Testing if correction of offset works in compress-function"""
+    # Arrange
+    path_in = retrieve_data("rtdc_data_hdf5_rtfdc.zip")
+    # Compress with and without offset correction
+    path_out = path_in.with_name("compressed.rtdc")
+    path_out_nc = path_in.with_name("compressed_nc.rtdc")
+
+    with new_dataset(path_in) as ds:
+        fl_max_old = ds["fl1_max"]
+        fl_offset = min(np.min(ds["trace"]["fl1_raw"]), 1)
```

Comment:
Please put the actual number here for future reference. I trust `min` and `np.min`, but `fl_offset` should be a static number for a test.

Reply:
While doing this for all tests, I realized that the fl_offset for the tdms-testfile is 0. Unfortunately this means that there is no suitable tdms-testfile. All other tdms-files are not suitable in some way or another (i.e. no trace data, or is a video-file, for which tdms2rtdc changes the length of the dataset leading to issues when comparing fl-max)
Do we have other tdms-testfiles I can use?